src/main/java/org/wikipedia/dataclient/mwapi/GeoSearchItem.java

```diff
@@ -0,0 +1,32 @@
+package org.wikipedia.dataclient.mwapi;
+
+import androidx.annotation.NonNull;
+
+import com.google.gson.annotations.SerializedName;
+
+public class GeoSearchItem {
+
+    @SuppressWarnings("unused") private String title;
```

Comment:
The `SuppressWarnings` annotation can be put at the top of the whole class, instead of for every individual field.